test/parallel/test-tls-client-auth.js

```diff
@@ -79,8 +79,10 @@ connect({
 }, function(err, pair, cleanup) {
   assert.strictEqual(pair.server.err.code,
                      'ERR_SSL_PEER_DID_NOT_RETURN_A_CERTIFICATE');
+  const expectedErr = common.hasOpenSSL32 ?
```

Comment:
This could be
```suggestion
  const expectedErr = common.hasOpenSSL(3, 2) ?
```
but `common.hasOpenSSL32` will still work as I kept the alias when I [added the more generic check function](https://github.com/nodejs/node/pull/53456)🙂.